python_processes/killableprocess.py

```diff
@@ -134,12 +134,12 @@ def setpgid_preexec_fn():
             subprocess.Popen.__init__(self, *args, **kwargs)
 
     if mswindows:
-        def _execute_child(self, args, executable, preexec_fn, close_fds,
-                           cwd, env, universal_newlines, startupinfo,
-                           creationflags, shell,
-                           p2cread, p2cwrite,
-                           c2pread, c2pwrite,
-                           errread, errwrite):
+        def _interal_execute_child(self, args, executable, preexec_fn, close_fds,
```

Comment:
No, the convention in python is that things with an underscore are not part of the public API, but as you know in python you *can* do anything, so this is just convention.